<<< is a undefined feature of POSIX

# Issue
In the file .config/newsboat/config line 39 is a reference to a undefined feature of POSIX so the dotfiles are not dash compatible, i noticed this after switching sh to dash
https://github.com/LukeSmithXYZ/voidrice/blob/master/.config/newsboat/config#L39
```
macro c set browser "xsel -b <<<"; open-in-browser; set browser linkhandeler
```
## __**ShellCheck**__
```
[julian@art ~]$ shellcheckj -
#!/bin/sh
xsel -b <<<a

In - line 2:
xsel -b <<<a
        ^-^ SC2039: In POSIX sh, here-strings are undefined.

For more information:
  https://www.shellcheck.net/wiki/SC2039 -- In POSIX sh, here-strings are und...
[julian@art ~]$ shellcheck -
#!/bin/dash
xsel -b <<<a

In - line 2:
xsel -b <<<a
        ^-^ SC2169: In dash, here-strings are not supported.

For more information:
  https://www.shellcheck.net/wiki/SC2169 -- In dash, here-strings are not sup...
[julian@art ~]$
```
## My idea
If for some reason something like a var with the url cloud be expanded i whould sujest this
```
macro c set browser "xsel -b <<EOF
$expansion
EOF"; open-in-browser ; set browser linkhandeler
```

## [DashAsBinSh](https://wiki.ubuntu.com/DashAsBinSh)
Points out
```
<<<
"Here strings" are similar to "here docs," and in most cases can easily be replaced with one.

E.g.

$ cat <<<"$HOME is where the heart is."
/home/ralph is where the heart is.
$
$ cat <<E
> $HOME is where the heart is.
> E
/home/ralph is where the heart is.
```
## Thanks in advance

--- a/.config/newsboat/config
+++ b/.config/newsboat/config
@@ -36,7 +36,7 @@ macro a set browser "tsp youtube-dl --add-metadata -xic -f bestaudio/best" ; ope
 macro v set browser "setsid -f mpv" ; open-in-browser ; set browser linkhandler
 macro w set browser "lynx" ; open-in-browser ; set browser linkhandler
 macro d set browser "dmenuhandler" ; open-in-browser ; set browser linkhandler
-macro c set browser "xsel -b <<<" ; open-in-browser ; set browser linkhandler
+macro c set browser "echo %u | xclip -r -sel c" ; open-in-browser ; set browser linkhandler
 macro C set browser "youtube-viewer --comments=%u" ; open-in-browser ; set browser linkhandler
 macro p set browser "peertubetorrent %u 480" ; open-in-browser ; set browser linkhandler
 macro P set browser "peertubetorrent %u 1080" ; open-in-browser ; set browser linkhandler

